source/agora/utils/Test.d

```diff
@@ -595,6 +598,27 @@ public struct TxBuilder
     private Transaction data;
 }
 
+
+    /***************************************************************************
+
+        Helper method for signing
+
+        Params:
+            txb = a `TxBuilder` instance
+            type = type of `Transaction`
+            data = data payload of `Transaction`
+
+        Returns:
+            The finalized & signed `Transaction`.
+
+    ***************************************************************************/
+
+public Transaction signTx (ref TxBuilder txb, TxType type = TxType.Payment,
```

Comment:
It seems the indentation of this (and the `spendable`) function is wrong. Shouldn't it be indented like the ddoc comment?

Reply:
Also I don't see where this function is used?

Reply:
> It seems the indentation of this (and the spendable) function is wrong. Shouldn't it be indented like the ddoc comment?

I simply followed the style of `spendable`, however that's not the case for other ddocs. Perhaps the ddoc for this and `spendable` should have no indentation ? (they are not part of the `TxBuilder` struct)

Reply:
Right then the docs should be unindented.

Reply:
> Also I don't see where this function is used?

"One of the goal here is to avoid having to edit every single use of it"
I am thinking of leaving the existing `sign`s, and this one can be used whenever one wants to use a not-well-known KeyPair. Open for suggestions.